modules/conflict_resolver/templates/menu_conflict_resolver.tpl

```diff
@@ -60,7 +60,7 @@
                             <input type="submit" name="filter" value="Show Data" class="btn btn-sm btn-primary col-xs-12"/>
                         </div>
                         <div class="col-sm-6 col-xs-12">
-                            <input type="button" name="reset" value="Clear Form" class="btn btn-sm btn-primary col-xs-12" onclick="location.href='main.php?test_name=conflict_resolver&reset=true'">
+                            <input type="button" name="reset" value="Clear Form" class="btn btn-sm btn-primary col-xs-12" onclick="location.href='?reset=true'">
```

Comment:
I think these should include the absolute {$baseurl}/conflict_resolver/ before the ? still (but they should be PrettyURLs instead of main.php references..)